disttask/framework/dispatcher/dispatcher_manager.go

```diff
@@ -143,11 +145,6 @@ func (dm *Manager) dispatchTaskLoop() {
 			logutil.BgLogger().Info("dispatch task loop exits", zap.Error(dm.ctx.Err()), zap.Int64("interval", int64(checkTaskRunningInterval)/1000000))
 			return
 		case <-ticker.C:
-			cnt := dm.getRunningTaskCnt()
-			if dm.checkConcurrencyOverflow(cnt) {
-				break
-			}
-
 			// TODO: Consider getting these tasks, in addition to the task being worked on..
 			tasks, err := dm.taskMgr.GetGlobalTasksInStates(proto.TaskStatePending, proto.TaskStateRunning, proto.TaskStateReverting, proto.TaskStateCancelling)
```

Comment:
How about initializing the metrics with these tasks when the DispatcherManager starts?

Reply:
It is precise.

Reply:
For DistDDLTaskStarttimeGauge, I think it's not necessary to initialize here because it doesn't change.
For DistDDLTaskGauge,  should I go through all the tasks in the tidb_global_task table?